lib/OrdinaryDiffEqFIRK/src/firk_tableaus.jl

```diff
@@ -111,6 +111,140 @@ function RadauIIA5Tableau(T, T2)
         e1, e2, e3)
 end
 
+struct BigRadauIIA5Tableau{T1, T2, Int}
+    T::AbstractMatrix{T1}
+    TI::AbstractMatrix{T1}
+    c::AbstractVector{T2}
+    γ::T1
+    α::AbstractVector{T1}
+    β::AbstractVector{T1}
+    #e::AbstractVector{T1}
+    num_stages::Int
+end
+
+function BigRadauIIA5Tableau(T1, T2, Int)
+    γ = convert(T1, 3.63783425274449573220841851357777579794593608687391153215117488565841871456727143375130115708511223004183651123208497057248238260532214672028700625775335843)
+    α = Vector{T1}(undef, 1)
+    β = Vector{T1}(undef, 1)
+    α[1] = convert(T1, 2.68108287362775213389579074321111210102703195656304423392441255717079064271636428312434942145744388497908174438395751471375880869733892663985649687112332242)
```

Comment:
also for the coefficients in Arrays, the `convert` isn't necessary (it happens automatically on the `setindex!`)